.eslintrc.yml

```diff
@@ -0,0 +1,18 @@
+env:
+  browser: true
+  es2021: true
+extends:
+  - eslint:recommended
+  - plugin:react/recommended
+  - plugin:json/recommended
+parserOptions:
+  ecmaFeatures:
+    jsx: true
+  ecmaVersion: 13
+  sourceType: module
+plugins:
+  - react
+rules: {
```

Comment:
this is weird.

Reply:
I run npx eslint .js file

Reply:
@krishguptadev can you write a more helpful comment?

Reply:
There is json in b/w of yml? Is this normal?